ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -194,6 +209,26 @@ open class PopupMenuController: DrawerController {
         view.isHidden = true
         return view
     }()
+    private lazy var searchView: UIView = {
+        let view = UIStackView()
+        let marginsSpacing = GlobalTokens.spacing(.size160)
```

Comment:
nit: I'd suggest renaming these variables since margin and spacing mean the same thing in this context.